andela/travel_tool_front
# #160729953 Rearrange column positions for destination and origin

#### What does this PR do?
* This PR rearrange the column positions for destination and origin 

#### Description of Task to be completed?
* Switch the input/column of destination and origin

#### How should this be manually tested?
* Clone this repo - `git clone https://github.com/andela/travel_tool_front.git`
* Switch to the repo directory - `cd travel_tool_front`
* Check out this branch - `git checkout ch-rearrange-travel-columns-160729953`
* Install dependencies and start the server - `make start`
* Set up the server by following this [guide](https://github.com/andela/travel_tool_back/pull/46)
* Navigate to localhost:3000 in a browser
* Login with your Andela mail
* Click on the `New Request` button

#### Any background context you want to provide?
* N/A

#### What are the relevant pivotal tracker stories?
* [#160729953](https://www.pivotaltracker.com/story/show/160729953)

#### Screenshots (if appropriate)
<img width="1411" alt="screen shot 2018-09-25 at 3 13 32 pm" src="https://user-images.githubusercontent.com/27569442/46070350-7a2e7700-c175-11e8-8478-4f53d011059e.png">

<img width="1237" alt="screen shot 2018-09-25 at 3 13 54 pm" src="https://user-images.githubusercontent.com/27569442/46070361-81558500-c175-11e8-974e-717808ee01d3.png">


#### Questions:
* N/A
